spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -58,16 +58,15 @@
         attach_images(
           Rails.root.join(
             'spec', 'fixtures',
-            'ial2_test_credential_barcode_attention_no_dob.yml'
+            'ial2_test_credential_barcode_attention_no_address.yml'
```

Comment:
This is curious to me, especially given my note above - is there a reason you changed from no dob to no address? I would have thought all pii would behave the same.

Reply:
I also found that this is the only place we use the `no_dob.yml`.

If we are switching to `no_address.yml` should we remove the un-used `no_dob`?

Reply:
while debugging this and trying to understand the issue, there was a period where I thought the underlying issue was something different and had created the address file along the way.  Having gotten to the bottom of the issue, I understand that it doesn't matter  whether  it's dob or address. I left the address as because I figured it'd be good to have more test files.